controllers/lrp_convergence_controller.go

```diff
@@ -307,7 +307,7 @@ func (h *LRPConvergenceController) ConvergeLRPs(ctx context.Context, logger lage
 			for _, ir := range dereferencedLRPKey.DesiredInternalRoutes {
 				internalRoutes = append(internalRoutes, internalroutes.InternalRoute{Hostname: ir.Hostname})
 			}
-			lrpUpdate := rep.NewLRPUpdate(dereferencedLRPKey.InstanceKey.InstanceGuid, *dereferencedLRPKey.Key, internalRoutes)
+			lrpUpdate := rep.NewLRPUpdate(dereferencedLRPKey.InstanceKey.InstanceGuid, *dereferencedLRPKey.Key, internalRoutes, nil)
```

Comment:
i was expecting there to be an additional section in LRP convergence here to ensure any metric tag updates events that are missed eventually converge on reps. Is this not needed?

Reply:
@geofffranks We're intending to add that convergence in a follow-up PR soon, we left it out of the scope for this PR.